docs/docs/en/guide/parameter/local.md

```diff
@@ -2,18 +2,27 @@
 
 ## Scope
 
-Parameters configured on the task definition page, the scope of this parameter is inside this task only. But if you configure according to [Refer to Parameter Context](context.md), it could pass to downstream tasks.
+Local parameters are used only for a single task node. Parameters configured on the task definition page, the scope of this parameter is inside this task only. But if you configure according to [Refer to Parameter Context](context.md), it could pass to downstream tasks.
```

Comment:
I think the previous version's description is better. We should keep it as it is.